src/recordpage.cpp

```diff
@@ -454,6 +461,12 @@ RecordPage::RecordPage(QWidget* parent)
         if (index != -1)
             ui->compressionComboBox->setCurrentIndex(index);
     }
+
+#ifdef __linux__
+    m_service = KService::serviceByDesktopName(QLatin1String("konsolepart"));
```

Comment:
`QStringLiteral`

Reply:
and move this down to the single place where you are using the service, I don't see a reason why this needs to be cached